sdk/containerregistry/Microsoft.Azure.ContainerRegistry/src/Customizations/AuthToken.cs

```diff
@@ -9,16 +9,16 @@
 namespace Microsoft.Azure.ContainerRegistry
 {
     /// <summary>
-    /// Simple authentication class for use with Token related api calls for determining scopes and other such things
+    /// Simple authentication class for use in clients used by Token classes.
     /// </summary>
-    public class TokenCredentials : ServiceClientCredentials
+    internal class TokenCredentials : ServiceClientCredentials
```

Comment:
Was this type public before? If it was, this is a breaking change.

Reply:
Yes it was Public earlier in this PR, however, this class is being introduced for the first time in this PR.